src/main/java/put/ai/se/jsontools/gui/FormatController.java

```diff
@@ -1,18 +1,34 @@
 package put.ai.se.jsontools.gui;
 
 import javafx.fxml.FXML;
-import javafx.scene.control.Alert;
-import javafx.scene.control.TextArea;
+import javafx.scene.control.*;
```

Comment:
try not to use * imports. Instead, use the "organize imports" functionality of the IDE.